### PR TITLE
Hability to change Endpoint

### PR DESCRIPTION
**Pull request by FernandoTBarros**
**Changes made:**

* Exposing the HASH_ENDPOINT variable to be able to change it outside of
the API (Mostly because of SSL issues, but also to facilitate the update
of the endpoint when the API of the server change)

### CODE REVIEW

**Comment by gegy1000** [comment]
If you add the ability to change the endpoint, you need the ability to change what connection to use.

**Comment by FernandoTBarros** [comment]
Actually it doesn't matter. That HttpURLConnection works both for Http and Https.
I'm using right now and it's working.

File: library/src/main/java/com/pokegoapi/util/hash/pokehash/PokeHashProvider.java [hunk 1]
```diff
@@ -40,7 +40,7 @@
  * This requires a key and is not free like the legacy provider.
  */
 public class PokeHashProvider implements HashProvider {
-	private static final String HASH_ENDPOINT = "https://pokehash.buddyauth.com/api/v121_2/hash";
+	public static String HASH_ENDPOINT = "https://pokehash.buddyauth.com/api/v122/hash";
```
Comment (gegy1000) [inline library/src/main/java/com/pokegoapi/util/hash/pokehash/PokeHashProvider.java]: Should be local to the provider class, as an optional parameter. Wasn't v122 removed?

Comment (FernandoTBarros) [inline library/src/main/java/com/pokegoapi/util/hash/pokehash/PokeHashProvider.java]: Don't Understand. What do you mean with "local to the provider class"? In the constructor? Or a get/set?
Indeed it was! I've looked in their channel this v122 sometime and though it was the last.. I'm gonna change it again to v121_2 according to https://pokehash.buddyauth.com/api/hash/versions

Comment (gegy1000) [inline library/src/main/java/com/pokegoapi/util/hash/pokehash/PokeHashProvider.java]: I mean it shouldn't be static.

**Pull request closed without merging**